package-lock.json

```diff
@@ -33762,7 +33762,7 @@
     },
     "packages/rbac": {
       "name": "@redhat-cloud-services/rbac-client",
-      "version": "2.2.4",
+      "version": "2.2.5",
```

Comment:
I double checked and I think this change is missing upstream (comes back after I remove it). @apinkert I wonder if we need to take a look if the publish step is including the package lock too when it checks in its commit?